bible_routes/v3/version_routes.py

```diff
@@ -82,6 +84,8 @@ async def add_version(
     user_group_ids = [group_id[0] for group_id in user_group_ids_query.all()]
```

Comment:
should we return a 422 if one of the groups in the list is not an actual group in the db? I do not have strong feelings either way but it maybe useful

Reply:
Yeah, I could go either way... let's maybe leave it for now.